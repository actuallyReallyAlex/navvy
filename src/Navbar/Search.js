import React, { Fragment, useState } from 'react'
import { Box, TextInput } from 'grommet'
import { Search } from 'grommet-icons'

const NavbarSearch = () => {
  const [searchValue, setSearchValue] = useState('')
  
  return (
    <Fragment>
      <Box margin={{ horizontal: 'small' }}>
        <Search />
      </Box>
      <Box align="center" direction="row" gap="small" width="50%">
        <TextInput
          margin
          onChange={e => setSearchValue(e.target.value)}
          placeholder="Search..."
          plain
          size="small"
          value={searchValue}
        />
      </Box>
    </Fragment>
  )
}

export default NavbarSearch
