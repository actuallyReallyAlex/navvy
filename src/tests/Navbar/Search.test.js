import React from 'react'
import Search from '../../Navbar/Search'
import { shallow } from 'enzyme'

const defaultProps = {}

describe('<Search />', () => {
  afterEach(() => {})

  test('Should render the <Search /> component.', () => {
    const component = shallow(<Search {...defaultProps} />)
    expect(component).toMatchSnapshot()
  })
})
