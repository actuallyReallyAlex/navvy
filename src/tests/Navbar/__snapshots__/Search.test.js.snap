// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Search /> Should render the <Search /> component. 1`] = `ShallowWrapper {}`;
